tests/baselines/reference/arrayFilter.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/arrayFilter.ts(7,12): error TS2345: Argument of type '(x: { name: string; }) => string' is not assignable to parameter of type '(value: { name: string; }, index: number, array: { name: string; }[]) => boolean'.
+  Type 'string' is not assignable to type 'boolean'.
+
+
+==== tests/cases/compiler/arrayFilter.ts (1 errors) ====
+    var foo = [
+       { name: 'bar' },
+       { name: null },
+       { name: 'baz' }
+    ]
+    
+    foo.filter(x => x.name); //should accepted all possible types not only boolean!
+               ~~~~~~~~~~~
+!!! error TS2345: Argument of type '(x: { name: string; }) => string' is not assignable to parameter of type '(value: { name: string; }, index: number, array: { name: string; }[]) => boolean'.

```

Comment:
What is this error doing here? It seems like this baseline file shouldn't exist.

The correct order of operations is:
 * Make your change
 * `jake runtests`
 * `jake baseline-accept`
 * `jake runtests` (this should always pass)
 * Then commit